cli/src/main/java/org/apache/iotdb/cli/AbstractCli.java

```diff
@@ -18,10 +18,12 @@
  */
 package org.apache.iotdb.cli;
 
+import static org.apache.iotdb.cli.utils.IoTPrinter.*;
+
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
-import java.io.PrintStream;
+import java.lang.Character.UnicodeScript;
```

Comment:
This import seems unnecessary. I think it should be remove.

Reply:
> This `import java.lang.Character.UnicodeScript` seems unnecessary. I think it should be removed.

Hi, already optimize imports